Library/Formula/cfv.rb

```diff
@@ -1,12 +1,19 @@
-require 'formula'
-
 class Cfv < Formula
-  homepage 'http://cfv.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/cfv/cfv/1.18.3/cfv-1.18.3.tar.gz'
-  sha1 '8f3361fb9c13fe2645f1df8c177c61459f50e846'
+  homepage "http://cfv.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/cfv/cfv/1.18.3/cfv-1.18.3.tar.gz"
+  sha256 "ff28a8aa679932b83eb3b248ed2557c6da5860d5f8456ffe24686253a354cff6"
 
   def install
-    man1.install gzip("cfv.1")
-    bin.install "cfv"
+    system "make", "prefix=#{prefix}", "mandir=#{man}", "install"
+  end
+
+  test do
+    mkdir "test"
```

Comment:
It uses the pathname for the ` .sha1 ` file - Since we don't assign static ` /tmp ` buildpaths, it needs to be run from inside another directory so we can correctly predict the file name. ` test ` just made sense as a name.